lib/phlex/rails/context.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+ApplicationControllerForPhlex = Class.new(ActionController::Base)
+
+module Phlex
+  module Rails
+    module Context
+      def render(partial = nil, **kwargs)
```

Comment:
I think capturing the first argument as the `partial` and soaking up `kwargs` as the locals is more consistent with the `render` view helper in Rails.